doc/sphinx/core-language.rst

```diff
@@ -0,0 +1,37 @@
+.. _core-language:
+
+=============
+Core language
+=============
+
+At the heart of the Coq proof assistant is the Coq kernel.  While
```

Comment:
Most of this explanation should be moved up to the main introduction.  And overlaps with my comments.

Reply:
While I have added a general introduction corresponding to your recommendation, I still think that having this introduction for the chapter is important. It's not so bad to repeat oneself as readers won't always read everything, and this gives an opportunity to give some more details compared to the general introduction.

Reply:
Agreed